packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -153,6 +153,16 @@ const VirtualListBaseFactory = (type) => {
 			 */
 			pageScroll: PropTypes.bool,
 
+			/**
+			 * The role for the list.
+			 * If unset, it defaults to the `list` of `role`
```

Comment:
We can remove this comment. The default is documented below so this is redundant.

Reply:
> We can remove this comment. The default is documented below so this is redundant.

If you mean to remove the role comment below, a lint error occurs.


```
/**
 * The role for the list.
 * If unset, it defaults to the `list` of `role`
 *
 * @type {String}
 * @default 'list'
 * @public
 */
 role: PropTypes.string,
```

`npm run lint` result:
```
@enact/moonstone@3.0.1 lint /Users/baekwoo.jung/work/enact/packages/moonstone
enact lint --strict

/Users/baekwoo.jung/work/enact/packages/moonstone/VirtualList/VirtualListBase.js
  799:35  warning  'role' is missing in props validation  enact/prop-types
```